openshift-tools/pkg/discover-etcd-initial-cluster/initial-cluster.go

```diff
@@ -168,7 +168,7 @@ func (o *DiscoverEtcdInitialClusterOptions) Run() error {
 		// If member is not yet part of the cluster print to stderr and retry.
 		if err != nil && !memberFound {
 			fmt.Fprintf(os.Stderr, "      %s\n#### sleeping...\n", err.Error())
-			time.Sleep(1 * time.Second)
+			time.Sleep(5 * time.Second)
```

Comment:
why wait longer?

Reply:
I want to wait 180s in general in the worse case, I figured 180 log entries might be a bit too much. So instead 36 probes with 5s deley. Does it make sense?

Reply:
I could also decrease to 2s which would give us ~100s for the startupProbe and a few more for the readiness one.

Reply:
Every second doesn't offend my sensibilities. I prefer 2 to 5 at the very least.

Reply:
okay, since the startupProbe waits for 180s, I gave 135s to this process and 45s to the etcd that runs after us. Does it make sense?